src/constants/numerical.js

```diff
@@ -36,5 +36,11 @@ module.exports = {
     ONEDAY: 86400000,
     ONEHOUR: 3600000,
     ONEMIN: 60000,
-    ONESEC: 1000
+    ONESEC: 1000,
+
+    /*
+     * For fast conversion btwn world calendars and epoch ms, the Julian Day Number
+     * of the unix epoch. From calendars.instance().newDate(1970, 1, 1).toJD()
+     */
+    EPOCHJD: 2440587.5
```

Comment:
If the whole contents of the [`if(calendar)`](https://github.com/plotly/plotly.js/pull/1230/files?diff=split#diff-b896a50c4f63f50409418cd75ff7d0aeR430) [blocks](https://github.com/plotly/plotly.js/pull/1230/files?diff=split#diff-b896a50c4f63f50409418cd75ff7d0aeR503) went into the component, you wouldn't need `EPOCHJD` in dates at all, would you?